lib/Rex/Virtualization/LibVirt/create.pm

```diff
@@ -444,7 +448,14 @@ __DATA__
 
    <% for my $disk (@{$::storage}) { %>
    <disk type="<%= $disk->{type} %>" device="<%= $disk->{device} %>">
-    <driver name="qemu" type="<%= $disk->{driver_type} %>"/>
+    <driver name="qemu" type="<%= $disk->{driver_type} %>"
+      <% if(exists $disk->{driver_cache}) { %>
+        cache="<%= $disk->{driver_cache} %>"
+      <% } %>
+      <% if(exists $disk->{driver_discard}) { %>
+        discard="<%= $disk->{driver_discard} %>"
+      <% } %>
+    />
```

Comment:
If I understand correctly, this is the first part of the story from the commit message (in other words, if we split the combined commit, it would be one part of the full story).

And it's adding support for `driver_cache` and `driver_discard` in the XML template. 

It _looks_ OK for that :+1: (but I didn't test)

Reply:
Split into a new commit.